hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -143,8 +143,12 @@ public AzureBlobFileSystemStore(URI uri, boolean isSecureScheme, Configuration c
     } catch (IllegalAccessException exception) {
       throw new FileSystemOperationUnhandledException(exception);
     }
+
+    LOG.trace("AbfsConfiguration init complete");
+
     this.userGroupInformation = UserGroupInformation.getCurrentUser();
     this.userName = userGroupInformation.getShortUserName();
+    LOG.trace("UGI init complete");
```

Comment:
seems superfluous.

Reply:
Agree on the multiple logging, but a recent debugging session with an internal client revealed that there could be multiple scenarios which could look like a hung state in ABFSDriver initialization. The intial debugging culprits included SSLDelegationContext instance and UGI init hitting a system error and failing to return the control back. 
The actual cause of the issue turned out to be a hang in the initialization of the custom token provider used in the user scenario. 
As a precaution adding a log line whenever the control goes out of the ABFS driver code. As it is too verbose, adding as TRACE logs. Please let me know if you still feel that

Reply:
well, if its in, print the username